import { connection } from "./database.js";
import {validator} from "./schemas.js";

export const asyncError = handlerFn = async (req, res, next) => {
    try {
        await handlerFn(req, res, next);
    } catch (err) {
        console.warn(err);
        res.status(500).send({
            message: "Internal Server Error"
        });
    }
};